src/components/RestaurantsListPage/RestaurantsListPage.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import RestaurantCard from '../RestaurantCard';
+import './RestaurantListPage.scss';
+import Loader from '../Loader';
+import Error from '../Error';
+
+const DEFAULT_ETA_RANGE = '20 - 30 min';
+
+export class RestaurantsListPage extends Component {
+  componentDidMount() {
+    const { loadRestaurants } = this.props;
+
+    loadRestaurants();
+  }
+
+  render() {
+    const {
+      restaurantsListData,
+      error,
+      isLoading,
+    } = this.props;
+
+    if (isLoading) {
```

Comment:
use logical operators

Reply:
I don't know how I can replace this piece of code with `return from render` to the logical operators

Reply:
`{isLoading && <Loader />}`

Reply:
I tried it and catch error 'Expected an assignment or function call and instead saw an expression'

Reply:
you can leave this

Reply:
ok